src/components/tasks/TasksContent.tsx

```diff
@@ -143,40 +144,48 @@ export const TasksContent = ({ dev }: { dev?: boolean }) => {
                 onInputChange={(value) => searchInputHandler(value)}
                 onClickSearchButton={searchButtonHandler}
             />
-            <div
-                className={classNames['status-tabs-container']}
-                data-testid="status-tabs-container"
-            >
-                <TabSection
-                    dev={dev}
-                    onSelect={(status: Tab) =>
-                        searchNewTasks(status, queryAssignees, queryTitle)
-                    }
-                    activeTab={selectedTab}
-                />
-            </div>
-            <div
-                className={classNames['status-select-container']}
-                data-testid="status-select-container"
-            >
-                <Select
-                    dev={dev}
-                    value={{
-                        label: getChangedStatusName(selectedTab),
-                        value: selectedTab,
-                    }}
-                    onChange={(selectedTaskStatus) => {
-                        if (selectedTaskStatus) {
-                            searchNewTasks(
-                                selectedTaskStatus.value as Tab,
-                                queryAssignees,
-                                queryTitle
-                            );
-                        }
-                    }}
-                    options={taskSelectOptions}
-                />
-            </div>
+            {dev !== true ? (
+                <>
```

Comment:
remove this fragment if not needed here

Reply:
This fragment is needed here as we have to wrap to <div> together